corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -82,6 +82,15 @@ FormplayerFrontend.reqres.setHandler('currentUser', function () {
     return FormplayerFrontend.currentUser;
 });
 
+FormplayerFrontend.reqres.setHandler('lastRecordedLocation', function() {
+    if (typeof sessionStorage.locationLat === "undefined") {
```

Comment:
I'm not familiar with this `sessionStorage` object though it seems to do the job well

We've been using `FormplayerFrontend.request('currentUser')` as our kind of all-purpose-dictionary - is there a reason you're using `sessionStorage` instead? Are there location lifecycle implications?

Reply:
It was just what I saw in examples online for doing something similar to what I'm doing (storing the browser location after it's retrieved). The lifecycle of the `sessionStorage` object is the duration of the window (which I think is a good lifecycle for caching the location). You can read more about it here if you want to make sure it seems ok to you, since I'm obviously not making super educated decisions on this https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage